### PR TITLE
Added a registration test

### PR DESCRIPTION
**Pull request by amolr**
Added a new test registration with tags update test

### CODE REVIEW

File: test/services/notificationHubsManagement/registration-tests.js [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+var should = require('should');
+var azure = require('azure');
```
Comment (amarzavery) [inline test/services/notificationHubsManagement/registration-tests.js]: `var azure = require('../../../lib/services/serviceBus/lib/servicebus');`

File: test/services/notificationHubsManagement/registration-tests.js [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+          location: namespaceLocation,
+          wnsCredential: {
+            packageSid:
+            'ms-app://s-1-15-2-1817505189-427745171-3213743798-2985869298-800724128-1004923984-4143860699',
+            secretKey: 'w7TBprR-THIS-IS-DUMMY-KEYAzSYFhp',
+            windowsLiveEndpoint: 'http://pushtestservice.cloudapp.net/LiveID/accesstoken.srf'
+          },
+          apnsCredential: {
+            token:
+            "MIGTAgEAMBMGByqGSM49AgEGCCqGSM49AwEHBHkwdwIBAQQgpVB15km4qskA5Ra5XvdtOwWPvaXIhVVQZdonzINh + hGgCgYIKoZIzj0DAQehRANCAASS3ek04J20BqA6WWDlD6 + xd3dJEifhW87wI0nnkfUB8LDb424TiWlzGIgnxV79hb3QHCAUNsPdBfLLF + Od8yqL",
```
Comment (amarzavery) [inline test/services/notificationHubsManagement/registration-tests.js]: is this a real token (this is a public repo)?

Comment (amolr) [reply test/services/notificationHubsManagement/registration-tests.js]: This is common set of Apple Notification Service credential we use across our team for testing. This is created for our team. Also, notificationhub-tests.js uses this token

File: test/services/notificationHubsManagement/registration-tests.js (hunk 1 above)
Comment (amolr) [reply test/services/notificationHubsManagement/registration-tests.js]: will make the change

**Comment by amarzavery** [comment]
You need to record these tests. The CI runs them in playback mode.

**Comment by RikkiGibson** [comment]
What's the status of this PR?

**Comment by RikkiGibson** [comment]
Please reopen if you'd like to do change or further discuss this PR

**Pull request closed without merging**